File: algorithm/src/main/java/baekjoon/p10156/Main.java
package baekjoon.p10156;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt(), sc.nextInt(), sc.nextInt()));
    }

    private static int solve(int k, int n, int m) {
        return Math.max(k * n - m, 0);
    }

}

File: algorithm/src/main/java/baekjoon/p10156/Main.kt
package baekjoon.p10156

fun main(args: Array<String>) {
    readLine()!!
            .split(" ")
            .map { it.toInt() }
            .let { println(solve(it[0], it[1], it[2])) }
}

private fun solve(k: Int, n: Int, m: Int): Int {
    return maxOf(k * n - m, 0)
}